src/Promitor.Agents.Scraper/Scheduling/ResourceDiscoveryGroupScrapingJob.cs

```diff
@@ -1,133 +0,0 @@
-﻿using System;
```

Comment:
Why is this being removed?

Reply:
No longer needed. Your CI pipelines treat warnings as errors for static code analysis so I had to implement every change ReSharper would point out.